### ISSUE TITLE
please help consolidate, ollama, LM studio Msty.app model paths, to avoid duplicate downloads

### ISSUE THREAD

**Issue by papyr** [Les-El/Ollm-Bridge#1]
Hello this is a great tool, but please allow us to use ollm-bridge at the main library or model management tool, since all these tools have different paths.

Problem is that LM studio downloads models to the cache folder, within other SUB folder per model. Howver, all the other apps, Ollama and Msty.app use the same folder..

**Comment by Les-El** [comment]
Hi! Thanks for the input. If I'm understanding your concerns correctly, you'd like to:

#### Use Ollm Bridge [as] the main library or model management tool
  * At this time, Ollm Bridge is a simple PowerShell script wrapped in an exe
  * Expanding capability to support Msty.app could be possible; perhaps by passing arguments and/or an interactive script 
  * Expanding into a robust management tool is beyond the project's scope at this time (but forks are invited!)
#### Avoid needing to download models more than once, wasting both time and storage capacity
  * This is exactly the use case that Ollm Bridge intends to solve
  * Ollm Bridge 0.6 creates SymbolicLinks leading to the binary files that Ollama uses to store language model files
  * These SymbolicLinks are placed within directories whose structures LM Studio is expecting
  * While some systems and applications (like WinDirStat) will treat these SymbolicLinks as if they are taking up many gigabytes of storage, each one actually occupies less than a kilobyte of space
 
Long story short, I'm going to take "Add Msty.app" as a feature request. Please feel free to add to, or to correct my comments. Thanks!

**Comment by ThomasEricB** [comment]
Hey dude, could you please look into this?

**Comment by Les-El** [comment]
Hi. Long story short, I don't think I'll be working on this soon. I've been focusing on accounting for the last year. And tbf, I've forgotten almost everything about this script.

The good news is that Beyonder 24B 5_K_M did the heavy lifting for this code, and code assistants have gotten much better since then. It has an "unlicense license," so anyone is free to have at it. Good luck!
